tutorials/preprocessing/plot_40_artifact_correction_ica.py

```diff
@@ -585,7 +585,57 @@
 # .. _`qrs`: https://en.wikipedia.org/wiki/QRS_complex
 # .. _`this EEGLAB tutorial`: https://labeling.ucsd.edu/tutorial/labels
 
+
+###############################################################################
+# Example: Compute ICA components on epochs
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# ICA is now fit to MEG epochs data instead of the raw data.
+# We assume that the non-stationary EOG artifacts have already been removed.
+# The sources matching the ECG are automatically found and displayed.
+#
+# .. note::
+#     This example does quite a bit of processing, so even on a
+#     fast machine it can take about a minute to complete.
```

Comment:
I think this could be rephrased but I do not have a good alternative. @drammock how is this usually communicated in tutorials or examples?